pyaedt/hfss.py

```diff
@@ -3635,47 +3681,47 @@ def assign_lumped_rlc_to_sheet(
                 point0 = axisdir[0]
                 point1 = axisdir[1]
             else:
-                point0, point1 = self.modeler.get_mid_points_on_dir(sheet_name, axisdir)
+                point0, point1 = self.modeler.get_mid_points_on_dir(assignment, axisdir)
 
-            if not sourcename:
-                sourcename = generate_unique_name("Lump")
-            elif sourcename in self.modeler.get_boundaries_name():
-                sourcename = generate_unique_name(sourcename)
+            if not name:
+                name = generate_unique_name("Lump")
+            elif name in self.modeler.get_boundaries_name():
+                name = generate_unique_name(name)
             start = [str(i) + self.modeler.model_units for i in point0]
             stop = [str(i) + self.modeler.model_units for i in point1]
             props = OrderedDict()
-            props["Objects"] = [sheet_name]
+            props["Objects"] = [assignment]
             props["CurrentLine"] = OrderedDict({"Start": start, "End": stop})
-            props["RLC Type"] = rlctype
-            if Rvalue:
+            props["RLC Type"] = rlc_type
+            if resistance:
                 props["UseResist"] = True
-                props["Resistance"] = str(Rvalue) + "ohm"
-            if Lvalue:
+                props["Resistance"] = str(resistance) + "ohm"
+            if inductance:
                 props["UseInduct"] = True
-                props["Inductance"] = str(Lvalue) + "H"
-            if Cvalue:
+                props["Inductance"] = str(inductance) + "H"
+            if capacitance:
                 props["UseCap"] = True
-                props["Capacitance"] = str(Cvalue) + "F"
-            return self._create_boundary(sourcename, props, "Lumped RLC")
+                props["Capacitance"] = str(capacitance) + "F"
+            return self._create_boundary(name, props, "Lumped RLC")
         return False
 
-    @pyaedt_function_handler()
-    def assign_impedance_to_sheet(self, sheet_name, sourcename=None, resistance=50, reactance=0, is_infground=False):
+    @pyaedt_function_handler(sheet_name="assignment", sourcename="name", is_infground="is_inifinite_ground")
+    def assign_impedance_to_sheet(self, assignment, name=None, resistance=50, reactance=0, is_inifinite_ground=False):
         """Create an impedance taking one sheet.
 
         Parameters
         ----------
-        sheet_name : str or list
+        assignment : str or list
             Name of the sheet or list to apply the boundary to.
```

Comment:
```suggestion
            One or more names of the sheets to apply the boundary to.
```
I'm not going to make the same suggestion on additional occurrences--but I think the suggested description works better given the two possible data types.